thinc/model.py

```diff
@@ -176,14 +176,14 @@ def get_dim(self, name: str) -> int:
         else:
             return value
 
-    def set_dim(self, name: str, value: int) -> None:
+    def set_dim(self, name: str, value: int, force: bool = False) -> None:
```

Comment:
```suggestion
    def set_dim(self, name: str, value: int, *, force: bool = False) -> None:
```

We should make this keyword only I think?